packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -99,6 +103,7 @@ const defaultProps = {
   onFocus: undefined,
   onSelect: undefined,
   required: false,
+  returnFocusOnClose: true,
```

Comment:
Could we reverse the naming of this to something like `disableButtonFocusOnClose` and have the default be `false`?

Reply:
It makes sense to change it to disableButtonFocusOnClose and default to false.

https://github.com/cerner/terra-framework/pull/699/commits/9a581a1cf70c8c205f4826e2b32959ccc2e2aea6